### PR TITLE
Mako templates created

### PR DESCRIPTION
**Pull request by gonczor**
image
image
https://kindergarten.stxnext.pl/browse/WDP-127

### CODE REVIEW

File: src/presence_analyzer/templates/main.html [hunk 1]
```diff
@@ -1,24 +1,25 @@
 <!doctype html>
+<%!
+    from flask.helpers import url_for
+    from flask.globals import request
+%>
```
Comment (piotrgrundas) [inline src/presence_analyzer/templates/main.html]: You shouldn't need it

**Review by piotrgrundas — CHANGES_REQUESTED** [review]

File: src/presence_analyzer/templates/main.html [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+    <meta charset=utf-8>
+    <title>Presence analyzer</title>
+    <meta name="keywords" content="" />
+    <meta name="description" content=""/>
+    <meta name="author" content="STX Next sp. z o.o."/>
+    <meta name="viewport" content="width=device-width; initial-scale=1.0">
 
-        <link href="/static/css/normalize.css" media="all" rel="stylesheet" type="text/css" />
-        <link href="/static/css/basic.css" media="all" rel="stylesheet" type="text/css"/>
+    <link href="/static/css/normalize.css" media="all" rel="stylesheet" type="text/css" />
```
Comment (piotrgrundas) [inline src/presence_analyzer/templates/main.html]: url_for

File: src/presence_analyzer/templates/main.html [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
+    <link href="/static/css/basic.css" media="all" rel="stylesheet" type="text/css"/>
```
Comment (piotrgrundas) [inline src/presence_analyzer/templates/main.html]: url_for

File: src/presence_analyzer/templates/main.html [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
 
-        <script src="/static/js/jquery.min.js"></script>
-        <script type="text/javascript" src="https://www.google.com/jsapi"></script>
-    {% endblock head %}
+    <script src="/static/js/jquery.min.js"></script>
```
Comment (piotrgrundas) [inline src/presence_analyzer/templates/main.html]: url_for

**Review by piotrgrundas — CHANGES_REQUESTED** [review]

File: src/presence_analyzer/templates/main.html (hunk 1 above)
Comment (gonczor) [reply src/presence_analyzer/templates/main.html]: I get this error when I delete that part: 
image

File: src/presence_analyzer/main.py [hunk 5]
```diff
@@ -2,7 +2,13 @@
...
-from flask import Flask
+import os
 
+from flask import Flask
+from flask_mako import MakoTemplates
+from mako.lookup import TemplateLookup
 
 app = Flask(__name__)  # pylint: disable=invalid-name
+LOOKUP = TemplateLookup(directories=[os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                                  'templates')])
```
Comment (piotrgrundas) [inline src/presence_analyzer/main.py]: style

File: src/presence_analyzer/templates/presence_start_end.html [hunk 6]
```diff
@@ -1,12 +1,16 @@
-{% extends 'main.html' %}
+<%inherit file="main.html"/>
 
-{% block additional_scripts %}
+<%!
+    from flask.helpers import url_for
+%>
```
Comment (piotrgrundas) [inline src/presence_analyzer/templates/presence_start_end.html]: ?

File: src/presence_analyzer/templates/mean_time_weekday.html [hunk 7]
```diff
@@ -1,12 +1,16 @@
-{% extends 'main.html' %}
+<%inherit file="main.html"/>
 
-{% block additional_scripts %}
+<%!
+    from flask.helpers import url_for
+%>
```
Comment (piotrgrundas) [inline src/presence_analyzer/templates/mean_time_weekday.html]: ?

**Review by piotrgrundas — CHANGES_REQUESTED** [review]

**Pull request merged**